lib/config/fileconf.go

```diff
@@ -799,6 +799,8 @@ type App struct {
 type Rewrite struct {
 	// Redirect is a list of hosts that should be rewritten to the public address.
 	Redirect []string `yaml:"redirect"`
+	// Headers is a list of extra headers to inject in the request.
+	Headers []string `yaml:"headers,omitempty"`
```

Comment:
how about making this a `map`, so you don't have to do any parsing?

Reply:
I thought about the same during implementation but decided to keep it as a list for a couple of reasons:
* If it was a map, scenario with adding multiple same headers would be kinda awkward.
* This was the UX requested in the original [ticket](https://github.com/gravitational/teleport/issues/4762#issuecomment-725049718).

Shouldn't be a big deal, the parsing is pretty trivial.